README.md

```diff
@@ -2,8 +2,41 @@
 
 A feature plugin for WordPress, which aims to create a new (better) way to manage and deliver notifications to the relevant audience.
 
-Want to get involved? Join our weekly office hours every Tuesday at 14:00 UTC in the [#feature-notifications](https://wordpress.slack.com/messages/C2K1C71FE) channel of the [Make WordPress Slack](https://make.wordpress.org/chat/).
+Want to get involved? Join our weekly office hours every Wednesday at 14:00 UTC in the [#feature-notifications](https://wordpress.slack.com/messages/C2K1C71FE) channel of the [Make WordPress Slack](https://make.wordpress.org/chat/).
 
 To get started, please take a look at our [project wiki](https://github.com/WordPress/wp-notify/wiki)
 
 See also [Trac ticket #43484](https://core.trac.wordpress.org/ticket/43484).
+
+## Development
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it’s:
+
+- Reporting a bug
+- Testing the plugin
+- Discussing the current state, features, improvements
+- Submitting a fix or a new feature
+
+We use GitHub to host code, to track issues and feature requests, as well as accept pull requests.
+By contributing, you agree that your contributions will be licensed under its GPLv2 License.
+
+Prerequisites:
+- [NodeJS](https://nodejs.org/en/download/)
+- [Composer](https://getcomposer.org/download/)
+- [wp-env](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/) (optional)
+- [Docker](https://docs.docker.com/get-docker/) (optional)
+
+Installation:
+```bash
+$ git clone https://github.com/WordPress/wp-feature-notifications.git
+$ cd wp-feature-notifications
+$ npm i && composer install
```

Comment:
Given we now have a `.nvmrc` file for node, is it worth adding a line about this?

Reply:
Thanks for updating, I'm conscious of new contributors who aren't as familiar with all these tools really. Maybe the line about requiring node could expand on it:

```markdown
- [NodeJS](https://nodejs.org/en/download/). We recommend using [nvm](https://github.com/nvm-sh/nvm) to ensure a compatible node version.
```